crates/editor_ui/src/inspector/events.rs

```diff
@@ -0,0 +1,40 @@
+use bevy::prelude::*;
+
+use bevy_egui::*;
+
+use crate::prelude::*;
+
+#[derive(Resource, Default)]
+pub struct EventTab;
+
+impl EditorTab for EventTab {
+    fn ui(&mut self, ui: &mut egui::Ui, _: &mut Commands, world: &mut World) {
+        inspect(ui, world);
+    }
+
+    fn title(&self) -> egui::WidgetText {
+        "Event".into()
```

Comment:
I would call it "Event Debugger", WDYT @rewin123 ?

Reply:
May be "Event sender"?